### PR TITLE
Web design

### PR DESCRIPTION
**Pull request by kensayo**
### Project 1: Positioning and floating elements - Milestones 5 & 6

In these Milestones we:

- Added Editor's Pick and Suggested Newsletter Sections
- Finished positioning of elements in article
- Deployed website through Github Pages
- Finished README

### CODE REVIEW

File: index.html
Comment (umairarshadbutt) [inline index.html]: Please add the description in `href` attribute

File: index.html
Comment (umairarshadbutt) [inline index.html]: Please add the description in `href` attribute

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @LoxiasCam and @kensayo 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

Kindly work on the suggested changes.
- [x] Please Fix the linter issues
- [x] Please add the description in `href` attribute
- [x] Please align the video in center
- [x] Update the README file as previous TSE explained
##
- **Note: Please do not merge PR before approving the project**
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work, team. 💻 👀 ✨ 
- Great job applying the necessary changes requested by the previous TSE. Your project is almost complete!

## However, there are some changes required 👀 🧰 🔨 🔧 
- In the bottom section of your page there is a broken image that is supposed to contain the NTY logo. Please fix this: 
image

### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## Hi @kensayo and @LoxiasCam  :wave: 

You've done well so far :+1:  and I commend you for that :clap: 
However, there are still some changes to be made before this project can be approved.

## REQUIRED

* [x] The text **SPACE & COSMOS** should be present in your navbar.
* [x] Add the contents of the `Suggested newsletter section`
* [x] Your Pull Request message should be a short description or summary of what you've done in that Pull Request, not the changes made after a TSE review.

### Make the requested changes and submit for another review.
## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Review by kobiyoyo — APPROVED** [review]
# Project Approved :thumbsup:
## Hello Team  ,  you have done a brilliant job with this project. :clap:
#### You can merge !!!

**Pull request merged**
